Digipost.Signature.Api.Client.Direct/ResponseUrls.cs

```diff
@@ -4,15 +4,29 @@ namespace Digipost.Signature.Api.Client.Direct
 {
     public class ResponseUrls
     {
+
+        private Uri _statusBaseUrl;
+
         public ResponseUrls(Uri redirectUrl, Uri statusBaseUrl)
         {
-            StatusBaseUrl = statusBaseUrl;
+            _statusBaseUrl = statusBaseUrl;
             Redirect = new RedirectReference(redirectUrl);
         }
 
         public RedirectReference Redirect { get; set; }
 
-        public Uri StatusBaseUrl { get; }
+        public Uri StatusBaseUrl
+        {
+            get
+            {
+                if (_statusBaseUrl == null)
+                {
+                    throw new InvalidOperationException("The status base URL is not available. This is most likely because status for this job is retrieved by polling. " +
+                                                        "Use DirectClient.GetStatusChange() to check for updated statuses for this job.");
```

Comment:
Replace string reference to method name with: _MethodBase.GetCurrentMethod().Name_